src/Moryx.AbstractionLayer/Activities/ActivityConstraintContext.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2025, Phoenix Contact GmbH & Co. KG
+// Licensed under the Apache License, Version 2.0
+
+using Moryx.AbstractionLayer.Constraints;
+using Moryx.AbstractionLayer.Processes;
+
+namespace Moryx.AbstractionLayer.Activities;
+
+/// <summary>
+/// Implementation of <see cref="IConstraintContext"/> used during process and activity handling
+/// </summary>
+public class ActivityConstraintContext : IConstraintContext
```

Comment:
Do we want to keep the interface, or would you count it as one of the interfaces that do not provide any benefits?

I mean it's empty and hides the one implementation that the ProcessEngine will always use.. 😅